source/administration/bulk-export-data.rst

```diff
@@ -0,0 +1,320 @@
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">team</td>
+      <td valign="middle">string</td>
+      <td>The name of the team this channel belongs to.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">name</td>
+      <td valign="middle">string</td>
+      <td>The name of the channel.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">display_name</td>
+      <td valign="middle">string</td>
+      <td>The display name for the channel.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">type</td>
+      <td valign="middle">string</td>
+      <td>The type of channel. Can have one the following values:<br>
+          <kbd>"O"</kbd> for a public channel.<br>
+          <kbd>"P"</kbd> for a private channel.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">header</td>
+      <td valign="middle">string</td>
+      <td>The channel header.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">purpose</td>
+      <td valign="middle">string</td>
+      <td>The channel purpose.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">scheme</td>
+      <td valign="middle">string</td>
+      <td>The name of the Scheme that should be applied to this team.</td>
+    </tr>
+  </table>
+  
+User object
+-----------
+.. raw:: html
+
+  <table width="100%" border="1" cellpadding="5px" style="margin-bottom:20px;">
+    <tr class="row-odd">
+      <th class="head">Field name</th>
+      <th class="head">Type</th>
+      <th class="head">Description</th>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">username</td>
+      <td valign="middle">string</td>
+      <td>The user’s username. This is the unique identifier for the user.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">email</td>
+      <td valign="middle">string</td>
+      <td>The user’s email address.</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">auth_service</td>
+      <td valign="middle">string</td>
+      <td>The authentication service used for this user account. <br>
+        <kbd>""</kbd> or not provided - password authentication.<br>
+        <kbd>"gitlab"</kbd> - GitLab authentication.<br>
+        <kbd>"ldap"</kbd> - LDAP authentication (E10 and E20)<br>
+        <kbd>"saml"</kbd> - Generic SAML based authentication (E20)<br>
+        <kbd>"google"</kbd> - Google OAuth authentication (E20)<br>
+        <kbd>"office365"</kbd> - Microsoft Office 365 OAuth Authentication (E20)</td>
+    </tr>
+    <tr class="row-odd">
+      <td valign="middle">auth_data</td>
+      <td valign="middle">string</td>
+      <td>The authentication data if <kbd>auth_service</kbd> is used. The value depends on the <kbd>auth_service</kbd> that is specified.<br>
+        The data comes from the following fields for the respective auth_services:<br>
+        <kbd>""</kbd> or not provided - must be omitted.<br>
```

Comment:
For the "" auth_service case, this will be an empty string. This actually makes me think we have a bug where what the exporter exports in this field for users using user/pass auth will be incompatible with the importer.

Reply:
Created a bug to validate: https://mattermost.atlassian.net/browse/MM-12670.